### ISSUE TITLE
Great work!

### ISSUE THREAD

**Issue by TheBricktop** [anetczuk/gdscript-protocol-generator#1]
Im currently trying to integrate opencv in python with godot to share data extracted from frames and the image frames itself.
The problem is that I need the fastest way to copy ( or share ) image data to the godot texture. Although python has a shared memory access, there is no way to retrieve the data back in godot.
I was thinking about using C# for this as there is a net lib with shared memory manager, but I didnt tested it yet.

**Comment by anetczuk** [comment]
This project uses [gdtype-python](https://github.com/anetczuk/gdtype-python) library which acts as translation layer between GDScript and Python data structures. If you want to use GdScript then IMO there is no more convenient (and still fast) way to communicate with external processes. Sure, the gdtype-python could be improved on Python side to achieve faster transfers or by implementing OpenCV specialized structures.
If you want to be as fast as possible then implementing Godot plugin in C++ and using OS layer is one possibility (then the same C++ code could be used in Python in form of Python binding).

**Comment by TheBricktop** [comment]
Yeah I thought about it, that the best way could be creating very basic interop library in rust or CPP and build it in gdextension. I'll probably get into it then.
